### PR TITLE
    WINDUP-918  Ruleset XSD: Reflect the enumerated attributes.

### PR DESCRIPTION
**Pull request by OndraZizka**
Comes with https://github.com/windup/windup-rulesets/pull/115

### CODE REVIEW

File: config-xml/schema/windup-jboss-ruleset.xsd [hunk 1]
```diff
@@ -234,14 +234,22 @@
...
+            <xs:attribute name="effort" type="xs:byte" use="optional" />
             <!-- <xs:attribute type="xs:string" name="message" use="optional"/> -->
         </xs:complexType>
     </xs:element>
 
+    <xs:simpleType name="severity-type">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="OPTIONAL"/>
+            <xs:enumeration value="MANDATORY"/>
+            <xs:enumeration value="POSSIBLE_ISSUES"/>
```
Comment (jsight) [inline config-xml/schema/windup-jboss-ruleset.xsd]: I think that we need to reconcile the language between "possible issues" and "potential issues" here.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1443/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/821/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1444/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/822/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1445/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/823/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1446/

**Comment by OndraZizka** [comment]
Comes with https://github.com/windup/windup-rulesets/pull/115

**Comment by OndraZizka** [comment]
retest

**Comment by OndraZizka** [comment]
retest

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/824/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1447/

**Comment by jsight** [comment]
Yeah, I found the attribute reordering to be a little strange too. Still +1 on the PR overall. :)

**Pull request merged**